python/tests/utils/test_manifest.py

```diff
@@ -232,8 +244,8 @@ def test_read_manifest_v2(generated_manifest_file_file_v2: str) -> None:
     assert manifest_list.manifest_length == 7989
     assert manifest_list.partition_spec_id == 0
     assert manifest_list.content == ManifestContent.DELETES
-    assert manifest_list.sequence_number is None  # inheritance
-    assert manifest_list.min_sequence_number is None  # inheritance
+    assert manifest_list.sequence_number == 3  # inherited
+    assert manifest_list.min_sequence_number == 3  # inherited
```

Comment:
The manifest list's sequence numbers aren't inherited. They are always statically set.